mod-source-record-manager-server/src/main/java/org/folio/services/flowcontrol/RawRecordsFlowControlServiceImpl.java

```diff
@@ -0,0 +1,109 @@
+package org.folio.services.flowcontrol;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.kafka.KafkaConsumerWrapper;
+import org.folio.verticle.consumers.consumerstorage.KafkaConsumersStorage;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import javax.annotation.PostConstruct;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static org.folio.rest.jaxrs.model.DataImportEventTypes.DI_RAW_RECORDS_CHUNK_READ;
+
+@Service
+public class RawRecordsFlowControlServiceImpl implements RawRecordsFlowControlService {
+  private static final Logger LOGGER = LogManager.getLogger();
```

Comment:
Will this return the root logger or the logger for this class?

Typically I would see `LogManager.getLogger(RawRecordsFlowcontrolServiceImpl.class)` or something like that.

Reply:
great!